pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -459,11 +459,17 @@ public void unloadNamespaceBundlesGracefully() {
     }
 
     public CompletableFuture<Optional<Topic>> getTopicIfExists(final String topic) {
-        return getTopic(topic, false /* createIfMissing */);
+        return getTopic(topic, false /* use allowAutoTopicCreation */);
```

Comment:
should this be `true`?

Reply:
Uh no. getTopicIfExists should never create a topic. That is the purpose of this method.

Reply:
oh I see. so the comment of the flag here is wrong, no?

it should be `/* createIfMissing */`

Reply:
Oh, good point. I goofed.